eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -66,7 +66,6 @@ parameters:
         eng/pipelines/coreclr/templates/perf-job.yml
         eng/pipelines/coreclr/templates/*-perf-*
         eng/pipelines/coreclr/templates/run-perf*
-        eng/pipelines/coreclr/templates/run-scenarios-job.yml
```

Comment:
Leaving this comment to remind myself to double check all the other paths which need updating.

Reply:
Has this been double checked, or is that something that makes enough sense to do in another PR?

Reply:
This has now been resolved